colcon_ros/task/ament_cmake/build.py

```diff
@@ -39,7 +41,9 @@ def add_arguments(self, *, parser):  # noqa: D102
         extension.set_context(context=self.context)
 
         # add a hook for each available shell
-        additional_hooks = []
+        additional_hooks = create_environment_hook(
+            'ament_current_prefix', Path(args.install_base),
+            self.context.pkg.name, 'AMENT_CURRENT_PREFIX', '', mode='prepend')
```

Comment:
This change doesn't make any sense. `AMENT_CURRENT_PREFIX` is supposed to contain a single path - the current prefix. Using it with `mode='prepend'` makes the resulting variable a list of paths which isn't intended.

Reply:
Does this mean I should add support for `replace` in `create_environment_hook()`?

Reply:
Please try the alternative proposal from ament/ament_package#85.

Reply:
I have confirmed your proposal fixes #67